templates/_header.html

```diff
@@ -22,6 +22,14 @@
             Store
           </a>
         </li>
+        <li
+          class="p-navigation__link {% if page_slug == 'about' %}is-selected{% endif %}"
```

Comment:
For now we are not going to link on the site.
We should add this at the end when all is good to go.